students/daniel_grubbs/lesson04/kata_fourteen.py

```diff
@@ -27,15 +29,36 @@ def print_header():
 
 
 def trigrams(content):
-    """Take the contents and create the trigrams."""
+    """Take the contents from file and create the trigrams."""
     for item in range(len(content) - 2):
-        buld_tupls = tuple(contents[item: item + 2])
```

Comment:
I see you removed this, but I also would make the comment don't bother with short hand variable names.  Only two more letters makes them real words.  Just spell it all out.